### PR TITLE
Expose /status client API for blockchain node monitoring

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Expose /status client API
- Tweak status structure and names
- Fix some broken tests

#216

### CODE REVIEW

File: p2p/index.js [hunk 1]
```diff
@@ -77,35 +79,74 @@ class P2pClient {
     this.intervalConnection = null;
   }
 
-  buildManagedPeersInfo() {
+  getManagedPeersInfo() {
```
Comment (liayoo) [inline p2p/index.js:82]: Just noticed, but is it still correct to say managed? How about getPeersInfo?

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks!

File: p2p/index.js (hunk 1 above)
Comment (platfowner) [reply p2p/index.js]: Just changed to use networkStatus.connectionInfo instead (they're dup info). Thanks.

File: p2p/index.js [hunk 2]
```diff
@@ -77,35 +79,66 @@ class P2pClient {
...
+  }
+
+  getStatus() {
+    const blockStatus = this.server.getBlockStatus();
+    return {
+      address: this.server.getNodeAddress(),
+      updatedAt: Date.now(),
+      lastBlockNumber: blockStatus.number,
+      networkStatus: this.getNetworkStatus(),
+      blockStatus: this.server.getBlockStatus(),
```
Comment (cshcomcom) [inline p2p/index.js:128]: minor: blockStatus?

**Comment by platfowner** [comment]
All tests passed. Now merging..

**Comment by minsulee2** [comment]
I was a bit late because of other meetings! LGTM!

File: p2p/index.js (hunk 2 above)
Comment (platfowner) [reply p2p/index.js:128]: Right. will be fixed in next CLs. Thanks.

**Pull request merged**